### PR TITLE
Remove statsd from boostrap & examples

### LINKED ISSUE #1086: Remove --statsd-enabled and related code
**Describe the solution you'd like**
There should be no code for now unsued flags, like `--statsd-enabled` and friends that is not needed since https://github.com/heptio/contour/pull/1085 (if/when it is merged).

I don't see a reason to not remove it, but would like some opinions first just in case (following the talk then code project recommendation :))

**Anything else you would like to add:** No

**Environment:**

- Contour version: master (before 0.13 release)

### PR DESCRIPTION
**Pull request by stevesloka**
Fixes #1086 by removing statsd sink from bootstrap config since it's no longer needed with Envoy >v1.10. Also removes `--statsd+` flags which are no longer needed.

Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: internal/envoy/config.go [hunk 1]
```diff
@@ -36,11 +36,11 @@ type ConfigWriter struct {
 	AdminPort int
 
 	// StatsAddress is the address that the /stats path will listen on.
```
Comment (rata) [inline internal/envoy/config.go]: contour metrics path is `/metrics`. Is the comment outdated or am I missing something? :)

File: internal/envoy/config.go [hunk 2]
```diff
@@ -36,11 +36,11 @@ type ConfigWriter struct {
...
-	// Defaults to 0.0.0.0 and is only enabled if StatsdEnabled is true.
+	// Defaults to 0.0.0.0
 	StatsAddress string
 
 	// StatsPort is the port that the /stats path will listen on.
```
Comment (rata) [inline internal/envoy/config.go]: Idem

File: internal/envoy/config.go (hunk 1 above)
Comment (rata) [reply internal/envoy/config.go]: Oh, this is the envoy part. This is now `/stats/prometheus`

Comment (davecheney) [reply internal/envoy/config.go]: yeah, its a bit confusing, this is the address of the listener we push to envoy for it to serve it's stats on.

The comment could probably be improved with ^^ that text.

Comment (stevesloka) [reply internal/envoy/config.go]: :+1:

**Pull request merged**